core/meterer/offchain_store.go

```diff
@@ -237,3 +237,79 @@ func (s *OffchainStore) GetRelevantOnDemandRecords(ctx context.Context, accountI
 
 	return prevPayment, nextPayment, nextDataLength, nil
 }
+
+func (s *OffchainStore) GetBinUsages(ctx context.Context, accountID string, binIndex uint32) (uint64, uint64, uint64, error) {
+	// Fetch the 3 bins start from the current bin
```

Comment:
3 is the minimum representative set of records of what the disperser is tracking, most significantly including the current bin being used, and the overflow bin that is located at current_bin_index+2

Reply:
The idea is that offchain_store will always return 3 records, because the disperser only track 4 in which one of them has became irrelevant. The `MinNumBins` is more for the user's accountants, if they prefer to track a longer history than 3, and their circular wrappings overwrite less often